gp2gp-translator/src/test/resources/xml/Quantity/pq_arbitrary_unit_uppercase_s.xml

```diff
@@ -0,0 +1,5 @@
+<value xmlns="urn:hl7-org:v3" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="urn:hl7-org:v3 ..\Schemas\RCMR_IN030000UK06.xsd" xsi:type="PQ" value="10" unit="1">
```

Comment:
Not feedback on your PR, but having this level of indirection, where your test data is outside of the test file really grinds my gears.  Especially when the test fixture is so small, like in this case.  Just put the XML into the Test file!